src/main/java/org/jabref/gui/collab/groupchange/GroupChangeDetailsView.java

```diff
@@ -0,0 +1,25 @@
+package org.jabref.gui.collab.groupchange;
+
+import javafx.scene.control.Label;
+
+import org.jabref.gui.collab.ExternalChangeDetailsView;
+import org.jabref.logic.l10n.Localization;
+
+public final class GroupChangeDetailsView extends ExternalChangeDetailsView {
+
+    public GroupChangeDetailsView(GroupChange groupChange) {
+        String labelValue = "";
+        if (groupChange.getGroupDiff().getNewGroupRoot() == null) {
+            labelValue = groupChange.getName() + '.';
+        } else {
+            labelValue = groupChange.getName() + ". " + Localization.lang("Accepting the change replaces the complete groups tree with the externally modified groups tree.");
+        }
```

Comment:
This seems odd. Is this a combined localization?
Maybe better something like `Localization.lang("%s. Accepting ... tree.", groupChange.getName())`?

Reply:
Yes, change name should be passed as a parameter to `lang()`. Not my code though, I only copy pasted it from the old code.

Reply:
Yes, you should be made parameterized. Other languages might have a different word order, so having parameters is more important

Reply:
@HoussemNasri as you are already on this, could you fix it in the places you merged it?

Reply:
*pasted it

Reply:
Yes, I'm on it